modules/database/src/ioc/db/dbLink.c

```diff
@@ -125,7 +127,9 @@ void dbInitLink(struct link *plink, short dbfType)
     if (dbfType == DBF_INLINK)
         plink->value.pv_link.pvlMask |= pvlOptInpNative;
 
-    dbCaAddLink(NULL, plink, dbfType);
```

Comment:
This is probably a side point, but I note that `dbfType` isn't actually used here. Is this an old API remnant that cannot easily be removed?

Reply:
> ... Is this an old API remnant that cannot easily be removed?

Yup.